src/mrnet/network/reaction_network.py

```diff
@@ -806,7 +928,7 @@ def parse_path(self, path):
         for step in path:
             if isinstance(step, int):
                 nodes.append(step)
-            elif "PR_" in step:
+            elif "PR_" in step:  # flag
```

Comment:
So first off, I don't totally remember when parse_path was introduced, and it seems weird to me that we have both parse_path and characterize_path. But also, I believe you said that PR solving was all working, and yet parse_path is still checking for "PR_"? Does that mean that parse_path isn't necessary to PR solving?